### PR TITLE
Make signed function attribute optional

### PR DESCRIPTION
**Pull request by akiramenai**
Solidity compiler might produce an abi w/o signed attribute for a
function. The patch makes it optional and the default value is False.

### CODE REVIEW

File: stdlib/abi_parser.py [hunk 1]
```diff
@@ -51,7 +54,7 @@ def convert_type (abi_type):
             errors.append ('Function %s may have 0 or 1 return values' % name)
 
     inputs = func['inputs']
-    signed = func['signed']
+    signed = func['signed'] if 'signed' in func else False
```
Comment (shd) [inline stdlib/abi_parser.py]: "signed" is not necessary anymore -- it's removed from ABI v.1., as far as I remember

File: stdlib/ton-sdk/messages.h [hunk 2]
```diff
@@ -4,7 +4,26 @@
 #include "tvm.h"
 
 #define HEADER_OR_C "define-ton-struct-header.inc"
+#include "datatype.inc"
```
Comment (shd) [inline stdlib/ton-sdk/messages.h]: Should we put the header into a separate file, if there is no datatype.c/.h?

File: llvm/lib/Target/TVM/TVMStackModel.cpp [hunk 3]
```diff
@@ -620,6 +620,10 @@ void TVMStackModel::modelInstructionExecution(MachineInstr &MI,
 
 void TVMStackModel::rewriteToSForm(MachineInstr &MI, std::string &PreTermStackString,
                                    Stack &TheStack) {
+  if (MI.isImplicitDef()) {
+    MI.eraseFromParent();
+    return;
+  }
```
Comment (shd) [inline llvm/lib/Target/TVM/TVMStackModel.cpp]: Maybe, move this to a separate commit?

Comment (mskvortsov) [reply llvm/lib/Target/TVM/TVMStackModel.cpp]: This has been already fixed by #14.
